src/utils.jl

```diff
@@ -1,9 +1,8 @@
 """
+sample!([rng,] a, k; exclude = ())
+
```

Comment:
Man, this is what I get for looking at this before caffeine. This is in the docstring, not the code. I get it. :)